dev/com.ibm.ws.javaee.ddmodel/resources/com/ibm/ws/javaee/internal/ddmodel/resources/DDModelMessages.nlsprops

```diff
@@ -70,21 +83,94 @@ invalid.href.prefix=CWWKC2260E: The href attribute value of the {2} element does
 
-missing.deployment.descriptor.namespace=CWWKC2264E: An error occurred while trying to determine the namespace of the {0} deployment descriptor on line {1}.
-missing.deployment.descriptor.namespace.explanation=An invalid namespace was specified in the root element of the deployment descriptor.
-missing.deployment.descriptor.namespace.useraction=Verify and correct the namespace in the root element of the deployment descriptor.
+# REMOVED:
+# invalid.deployment.descriptor.version=CWWKC2263E: The version {2} attribute specified in the {0} deployment descriptor on line {1} is not valid.
+# invalid.deployment.descriptor.version.explanation=An invalid version was specified in the root element of the deployment descriptor.
+# invalid.deployment.descriptor.version.useraction=Verify and correct the version number in the root element of the deployment descriptor.
+
+# REPLACED BY:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a version attribute value within the deployment descriptor
+#    {3} is the highest version version attribute value which is supported by the features currently provisioned by the server configuration
+# -----------------------------------------------------------------------
+unprovisioned.descriptor.version=CWWKC2263E: The deployment descriptor {0}, at line {1}, specifies version {2}, which is higher than the current provisioned version {3}.
+unprovisioned.descriptor.version.explanation=The version which was specified for the deployment descriptor is not supported by the features currently provisioned by the server configuration.
+unprovisioned.descriptor.version.useraction=Update the server configuration to a feature level which supports the version.
+
+# REMOVED:
+# missing.deployment.descriptor.namespace=CWWKC2264E: An error occurred while trying to determine the namespace of the {0} deployment descriptor on line {1}.
+# missing.deployment.descriptor.namespace.explanation=An invalid namespace was specified in the root element of the deployment descriptor.
+# missing.deployment.descriptor.namespace.useraction=Verify and correct the namespace in the root element of the deployment descriptor.
+
+# REPLACED BY:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a namespace value which is required to be present in the deployment descriptor
+# -----------------------------------------------------------------------
+missing.descriptor.namespace=CWWKC2264E: The deployment descriptor {0}, at line {1}, does not have required namespace {2}.
```

Comment:
Change "deployment descriptor {0}" to "{0} deployment descriptor"

Reply:
Updated.